packages/modular-scripts/package.json

```diff
@@ -83,14 +83,14 @@
     "html-webpack-plugin": "5.5.0",
     "is-ci": "2.0.0",
     "is-root": "2.1.0",
-    "jest": "^29.0.0",
-    "jest-circus": "^29.0.0",
-    "jest-cli": "^29.0.0",
-    "jest-config": "^29.0.0",
-    "jest-environment-jsdom": "^29.4.2",
-    "jest-runner-eslint": "1.1.0",
+    "jest": "^29.5.0",
```

Comment:
I updated Jest initially to use the fix suggested [here](https://github.com/facebook/jest/issues/11956).
Ended up not using the fix, as it slowed tests down too much, but keeping the updated Jest version